example/lstm_master.py

```diff
@@ -1,6 +1,9 @@
 import numpy
 import time
 
+import sys
+sys.path.append('..')
```

Comment:
So we should doc that they must be run from this directory.
Le 11 janv. 2016 14:52, "abergeron" <notifications@github.com> a écrit :

> In example/lstm_master.py
> <https://github.com/mila-udem/platoon/pull/11#discussion_r49369227>:
>
> > @@ -1,6 +1,9 @@
> >  import numpy
> >  import time
> >
> > +import sys
> > +sys.path.append('..')
>
> The examples are not in the same packages from python's point of view.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/platoon/pull/11/files#r49369227>.
>